app/src/main/kotlin/at/bitfire/davdroid/ui/AppSettingsScreen.kt

```diff
@@ -628,6 +643,21 @@ private fun PushDistributorSelectionDialog(
                         )
                     )
                 }
+
+                item {
+                    Text(
+                        text = buildAnnotatedString {
+                            pushStyle(
+                                SpanStyle(color = MaterialTheme.colorScheme.primary, textDecoration = TextDecoration.Underline)
+                            )
+                            pushLink(
+                                LinkAnnotation.Url("https://unifiedpush.org/users/faq/#q-is-unifiedpush-secure")
```

Comment:
Not sure whether it's a good idea to put an external link here. Maybe we could just stick with the text message for now and link to our own documentation once push is not experimental anymore ... but idk

Reply:
I see you are writing the docs in https://github.com/bitfireAT/davx5-manual/pull/46 already. You could mention the encryption there and I would definetly link there instead.

Reply:
Mmmmh okay, then this should wait for https://github.com/bitfireAT/davx5-manual/pull/46

Reply:
Don't know whether we should link it at all, but I have changed it for now so that it links to the new manual page